server/routes/probationPractitionerReferrals/probationPractitionerReferralsController.ts

```diff
@@ -129,7 +130,12 @@ export default class ProbationPractitionerReferralsController {
         this.communityApiService.getConvictionById(crn, sentReferral.referral.relevantSentenceId),
         this.assessRisksAndNeedsService.getSupplementaryRiskInformation(sentReferral.supplementaryRiskId, accessToken),
         this.assessRisksAndNeedsService.getRiskSummary(crn, accessToken),
-        this.communityApiService.getStaffDetails(sentReferral.sentBy.username),
+        new Promise<DeliusStaffDetails | null>(resolve => {
```

Comment:
nit: perhaps it would be easier to just have the service method return a null value on 404?

Reply:
this way we wouldn't have to duplicate this code in the other controller too.

Reply:
Yes, I realised that when I pushed .
I'm just writing one more commit for an additional test